### PR TITLE
Proposed Buchla pulses update

### PR DESCRIPTION
**Pull request by mathomas**
Hey there.  I have a 2OC that I was itching to put to work as a clock divider (etc) in my small Buchla setup.  However, I found that pulses weren't sufficient to trigger (at least) the 281e.

What you find here seems to get pulses up to about 10v in my scope (and does successfully trigger the 281e).  I've scoped most, if not all, the other pulse/gate oriented things and they look pretty good.  I have not tested musically, but I have pretty high confidence at this point.

I'm not sure if you already have a much better way of approaching it, but if not, at least what I'm doing is a possibility?

### CODE REVIEW

**Comment by mathomas** [comment]
Note:  I have another branch going regarding CVs but that's a bit trickier.  I've successfully scaled them on the high-side to net 10v peak.  But I haven't yet figured out how to properly offset to avoid negative/bipolar results from LFOs, etc.

**Comment by Chysn** [comment]
Thank you for your contribution. This looks good, and I'll probably integrate it shortly.

Please note that negative values should be prevented in OC_DAC.h via the setting of kOctaveZero. I've got a 2OC here, so I'll test this this weekend.

File: software/o_c_REV/OC_options.h [hunk 1]
```diff
@@ -12,7 +12,7 @@
 /* ------------ uncomment for use with Northernlight cOC program card: ------------------------------  */
 //#define BUCHLA_cOC
 /* ------------ uncomment for use with Northernlight 4U / cOC or 2OC: -------------------------------  */
-//#define BUCHLA_4U
+#define BUCHLA_4U
```
Comment (Chysn) [inline software/o_c_REV/OC_options.h]: This option should remain commented out for the main codebase.

Comment (mathomas) [reply software/o_c_REV/OC_options.h]: Absolutely.  Oversight on my part.

**Comment by mathomas** [comment]
> Thank you for your contribution. This looks good, and I'll probably integrate it shortly.
> 
> Please note that negative values should be prevented in OC_DAC.h via the setting of kOctaveZero. I've got a 2OC here, so I'll test this this weekend.

Sounds good.  I did some testing with my CV adjustment code (obviously very similar to the pulse adjustment here).  On intrinsically unipolar stuff like EGs, things look good.  Unfortunately, I do see basically “half rectified” waves, say with the LFOs, which supports what you say WRT bipolar.  On Buchla, I’d need to bias/offset the waves to fit into 0-10v.  Haven’t figured that out yet.

**Pull request merged**